IonDotnet/Internals/Tree/SystemTreeReader.cs

```diff
@@ -0,0 +1,323 @@
+﻿using IonDotnet.Tree;
+using IonDotnet.Tree.Impl;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Text;
+
+namespace IonDotnet.Internals.Tree
+{
+    internal class SystemTreeReader : IIonReader
+    {
+
+        protected readonly IIonValue _value;
```

Comment:
This variable doesn't seem to be used anywhere

Reply:
Removed.